plugins/wazo-yealink/v84/tests/test_common.py

```diff
@@ -0,0 +1,429 @@
+            'locale': 'en_US',
+            'funckeys': {},
+            'sip_proxy_ip': '1.1.1.1',
+            'sip_lines': {'1': {'number': '5888'}}
+        }
+        v84_plugin._tpl_helper.get_dev_template.return_value = 'template'
+        v84_plugin.configure(device, raw_config)
+        v84_plugin._tpl_helper.get_dev_template.assert_called_with('805ec0d57d72.cfg', device)
+        v84_plugin._tpl_helper.dump.assert_called_with(
+            'template', raw_config, 'test_dir/var/tftpboot/805ec0d57d72.cfg', 'UTF-8'
+        )
+
+    @patch('os.remove')
+    def test_deconfigure(self, mocked_remove, v84_plugin):
+        v84_plugin.deconfigure({'mac': '80:5e:c0:d5:7d:72'})
+        mocked_remove.assert_called_with('test_dir/var/tftpboot/805ec0d57d72.cfg')
+
+    @patch('v84.common.logger')
+    def test_deconfigure_no_file(self, mocked_logger, v84_plugin):
+        v84_plugin.deconfigure({'mac': '00:00:00:00:00:00'})
+        message, exception = mocked_logger.info.call_args.args
+        assert message == 'error while removing file: %s'
+        assert isinstance(exception, FileNotFoundError)
+
+    @patch('v84.common.synchronize')
+    def test_synchronize(self, provd_synchronize, v84_plugin):
+        device = {'mac': '80:5e:c0:d5:7d:72'}
+        v84_plugin.synchronize(device, {})
+        provd_synchronize.standard_sip_synchronize.assert_called_with(device)
+
+    def test_get_remote_state_trigger_filename(self, v84_plugin):
+        assert v84_plugin.get_remote_state_trigger_filename({}) is None
+        assert v84_plugin.get_remote_state_trigger_filename(
+            {'mac': '80:5e:c0:d5:7d:72'}
+        ) == '805ec0d57d72.cfg'
+
+    def test_sensitive_file(self, v84_plugin):
+        assert v84_plugin.is_sensitive_filename('patate') is False
+        assert v84_plugin.is_sensitive_filename('805ec0d57d72.cfg') is True
+
+    def test_add_country_and_lang(self, v84_plugin):
+        raw_config = {'locale': 'en_US'}
+        v84_plugin._add_country_and_lang(raw_config)
+        assert raw_config['XX_country'] == 'United States'
+        assert raw_config['XX_lang'] == 'English'
+        assert raw_config['XX_handset_lang'] == '0'
+        raw_config = {'locale': 'ja_JP'}
+        v84_plugin._add_country_and_lang(raw_config)
+        assert raw_config == {'locale': 'ja_JP'}
+
+    def test_checks(self, v84_plugin):
+        with pytest.raises(RawConfigError):
+            v84_plugin._check_config({})
+
+        with pytest.raises(Exception):
+            v84_plugin._check_device({})
+
+    @patch('v84.common.plugins')
+    def test_phonebook_url(self, provd_plugins, v84_plugin):
+        raw_config = {
+            'config_version': 0,
+            'X_xivo_phonebook_ip': '1.1.1.1',
+        }
+        v84_plugin._add_xivo_phonebook_url(raw_config)
+        provd_plugins.add_xivo_phonebook_url.assert_not_called()
+        expected_url = 'http://1.1.1.1/service/ipbx/web_services.php/phonebook/search/?name=#SEARCH'
```

Comment:
This is the old version of the phonebook URL

Reply:
Oh ok. Isn't that the point though? This was in the old version of the function for older systems, right?
Do we want to keep it or remove it with the other one?